packages/components/forms/src/BaseCheckbox/BaseCheckbox.styles.ts

```diff
@@ -25,14 +25,20 @@ const getHelpTextStyle = ({ size, type }) => {
   return {
     marginLeft: `calc(${inputWidth} + ${tokens.spacingXs})`,
     marginTop: 0,
+    fontSize: density === 'high' && tokens.fontSizeS,
+    lineHeight: density === 'high' && tokens.lineHeightS,
```

Comment:
We could define the default (low) value here too, based on the design spec, as you did for the `BaseCheckbox` Text component.

Reply:
good shout, couldn't hurt eh?

Reply:
done :)